hydra-cardano-api/src/Hydra/Cardano/Api/PlutusScript.hs

```diff
@@ -30,11 +36,10 @@ toLedgerScript ::
   PlutusScript lang ->
```

Comment:
Could: add `HasCallStack` to find the call site in case we ever use `PlutusV3` before updating this.